resources/ory/Chart.yaml

```diff
@@ -1,5 +1,6 @@
 apiVersion: v1
-appVersion: "1.0"
```

Comment:
With ory we in general want to put it 1:1 as much as possible with upstream chart